test/unit/test_builder.py

```diff
@@ -115,8 +142,9 @@ def compose_status(self, request, uri, response_headers):
         ireqs = compose["request"]["image_requests"]
         result = {
             "status": compose["status"],
-            "koji_task_id": compose["request"]["koji"]["task_id"],
```

Comment:
So task ID is no longer returned by composer? Was it not used or what is the reason?